test/test-plugin-loader.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2017 Google Inc. All Rights Reserved.
+ * Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
While you're changing the date, change 'Inc' to 'LLC' as well?

Reply:
Done